docs/LogDownloadResponse.md

```diff
@@ -0,0 +1,9 @@
+# LogDownloadResponse
+
+## Properties
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
```

Comment:
Strange that this file exists with no additional information. Looks like `KeytabDownloadUploadResponse.md` (which is also used for a file download) is more of the same. 
I don't think we need to do anything for this, but it would be nice if we could provide more information on what the expected response is for future releases.